content/en/platform/corda/4.12/enterprise/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+date: '2020-04-07T12:00:00Z'
+description: "Documentation for the 4.12 Enterprise Edition release of Corda"
+menu:
+  versions:
+    weight: -800
```

Comment:
Think this still needs to be changes so it appears above 4.11 in the drop-down menu. Looks like there is a simialr change needed for the Community file.

Reply:
Updated